css-fonts-5/Overview.bs

```diff
@@ -168,24 +168,49 @@ For: @font-face
 Initial: normal
 </pre>
 
+<pre class='descdef'>
+Name: size-adjust
+Value: normal | <<percentage>>
+For: @font-face
+Initial: normal
+</pre>
+
 The 'ascent-override', 'descent-override', and 'line-gap-override' descriptors define the
 <a spec="CSS-INLINE-3">ascent metric</a>, <a spec="CSS-INLINE-3">descent metric</a> and
-<a spec="CSS-INLINE-3">line gap metric</a> of the font, respectively.
+<a spec="CSS-INLINE-3">line gap metric</a> of the font, respectively. The `size-adjust`
+descriptor defines a scalar by which glyph outlines and metrics from this font are multiplied.
 
 When the descriptor value is 'normal', the corresponding metric value is obtained as-if the
 descriptor was absent from the <code>@font-face</code> block.
 
-Note: User agents may draw data from different places from the font file as the metric values,
+Note: User agents may gather data from different places from the font file as the metric values,
 which results in different text layouts.
 
 For the 'ascent-override', 'descent-override' and 'line-gap-override' descriptors, when the
 descriptor value is a percentage, the corresponding metric value is resolved as the
 given percentage multiplied by the used font size. Negative values are invalid at parse time.
 
+For 'size-adjust', when the descriptor value is a percentage, all metrics values of this font,
+including glyph advance sizes, are resolved as the given percentage multiplied by the used font
+size. When glyphs are visibly drawn, the visual size of the glyphs is also multiplied by the
+percentage.
+
+None of these descriptors affect the calculation of
+<a href="https://drafts.csswg.org/css-values-4/#font-relative-lengths">Font-relative Lengths</a>.
+None of these descriptors affect the behavior of any of the values of 'line-height' other than
+'normal'. The behavior of <code>line-height: normal</code> may be affected by all of these
+descriptors. 'size-adjust' does not affect the computed font size of any element.
+
+When 'size-adjust' is applied in conjunction with 'ascent-override', 'descent-override',
+or 'line-gap-override' in the same @font-face block, the relevant metrics are multiplied by both
+percentages.
```

Comment:
This is a bit misleading, I think. The `{ascent|descent|line-gap}-override` descriptors *don't* multiply "the relevant metrics" by the given percentage; they *replace* the metrics derived from the font with metrics computed from the used font size.